package.json

```diff
@@ -40,27 +40,28 @@
 	"dependencies": {
 		"@cucumber/cucumber": "^7.3.1",
 		"@cucumber/pretty-formatter": "^1.0.0-alpha.1",
-		"axios": "^0.21.1",
+		"axios": "^0.21.4",
```

Comment:
Only a hint, for package updates it is better to do it in a additional pr. You can use npm ci instant of npm i to install the packages without updates. But the updates do not show critical, also it is fine for this pr.